src/smart-components/Thread/context/utils.ts

```diff
@@ -25,15 +25,14 @@ export const getParentMessageFrom = (message: UserMessage | FileMessage): UserMe
 export const isParentMessage = (message: UserMessage | FileMessage): boolean => {
   return (
     message?.parentMessage === null
-    && message?.parentMessageId === null
-    && message?.threadInfo !== null
+    && !message?.parentMessageId
   );
 };
 
 export const isThreadMessage = (message: UserMessage | FileMessage): boolean => {
   return (
     message?.parentMessage !== null
-    && message?.parentMessageId !== null
+    && message?.parentMessageId
```

Comment:
add unit test before merge pls

Reply:
It's sure that the type of parentMessaeId is number, so I wanted to check if it's 0 or more